CODEOWNERS

```diff
@@ -0,0 +1,35 @@
+*                                                               @costinm @geeknoid @howardjohn @linsun @rshriram
+bin/                                                            @costinm @douglas-reid @mandarjog @sdake
+galley/                                                         @geeknoid @ozevren
+install/                                                        @costinm @sdake @linsun @rshriram @howardjohn @mandarjog @morvencao @ostromart
+install/kubernetes/helm/istio/charts/galley/                    @cmluciano @geeknoid @ozevren @ayj
+install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/    @ostromart
```

Comment:
This will change semantics a lot, now only Martin can modify it instead of everyone in `installer` and `*`, so ~10 owners to 1. 

Are you planning to modify this based on the new maintainers lists?

Reply:
Shoot, you're right. My  intent was to keep the same ownership, but I didn't propagate all the upper directory owners to the lower directories. I'll fix that.

Once we're on CODEOWNERS, then we'll update to the new maintainer data.

Reply:
I considered writing a quick script to do the propagation but I figured it would be easy to just use the new maintainers data and simplify it to be less fine-grained then we have today

Reply:
OK, this should be a lot closer to matching the original.

Reply:
Still seems the same to me, did you not push the change?